rdr_service/dao/genomics_dao.py

```diff
@@ -3178,6 +3218,19 @@ def from_client_json(self):
         pass
 
 
+class GenomicGCROutreachEscalationNotifiedDao(BaseDao, GenomicDaoMixin):
+
+    def __init__(self):
+        super(GenomicGCROutreachEscalationNotifiedDao, self).__init__(
+            GenomicGCROutreachEscalationNotified, order_by_ending=['id'])
+
+    def get_id(self, obj):
+        pass
+
+    def from_client_json(self):
+        pass
+
+
```

Comment:
Instead of us having to create a new one of these each time, I created a default dao called `GenomicDefaultBaseDao` so maybe we can utilize this?

Reply:
Yes, I've updated the PR to use GenomicDefaultBaseDao.